Sources/AEPOptimize/Optimize.swift

```diff
@@ -32,6 +32,13 @@ public class Optimize: NSObject, Extension {
     /// Dispatch queue used to protect against simultaneous access of our containers from multiple threads
     private let queue: DispatchQueue = .init(label: "com.adobe.optimize.containers.queue")
 
+    /// a dictionary containing the update event IDs and corresponding errors as received from Edge SDK
+    private var _errorUpdateRequestEventIds: [String: AEPOptimizeError] = [:]
+    private var errorUpdateRequestEventIds: [String: AEPOptimizeError] {
```

Comment:
nit: is it ok if we call this `_updateRequestEventIdsError` instead to be symmetrical with `_updateRequestEventIdsInProgress`?

Reply:
Done